crates/router/src/lib.rs

```diff
@@ -20,10 +20,20 @@ use ya_sb_util::PrefixLookupBag;
 mod dispatcher;
 
 lazy_static! {
-    pub static ref GSB_PING_TIMEOUT: u64 = env::var("GSB_PING_TIMEOUT")
-        .unwrap_or("5".into())
+    static ref RAW_GSB_PING_TIMEOUT: u64 = env::var("GSB_PING_TIMEOUT")
+        .unwrap_or(20.to_string())
         .parse()
         .unwrap();
+    // After GSB_PING_TIMEOUT without reply the connection will disconnect ( default = 20s )
+    pub static ref GSB_PING_TIMEOUT: Duration = Duration::seconds(*RAW_GSB_PING_TIMEOUT as i64);
+
+    // NOTE: Check and try intervals should be lower than ping timeout to avoid race conditions
+    // Try to send a ping to each connection at this interval ( default = 5s )
+    pub static ref GSB_PING_TRY_INT: Duration = *GSB_PING_TIMEOUT / 4;
```

Comment:
INT is ambiguous here as a short name of INTERVAL. It rather recalls INTEGER in IT world.

Reply:
i thought the type "Duration" would help distinguish from `INTEGER`, but can change it :)